events/syscall/remove_bulk_data_from_disk.go

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package syscall
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/falcosecurity/event-generator/events"
+)
+
+var _ = events.Register(RemoveBulkDataFromDisk)
+
+func RemoveBulkDataFromDisk(h events.Helper) error {
+	// Creates temporary data for testing, avoiding critical file deletion.
+	tmpDir, err := os.MkdirTemp(os.TempDir(), "created-by-falco-event-generator")
```

Comment:
```suggestion
	tmpDir, err := os.MkdirTemp(os.TempDir(), "created-by-falco-event-generator")
	defer os.RemoveAll(tmpDir) // clean up
```

Reply:
@leogr we again defer os.RemoveAll(tmpDir) as we already removing it with shred command in the same code at last
https://github.com/falcosecurity/event-generator/blob/da107c694de5b40061d4419fd0f4ca7c645ae89b/events/syscall/remove_bulk_data_from_disk.go#L42-L44

Reply:
I did not consider that, you're likely right.

Just one doubt: what happens if `shred` fails? :thinking:

Reply:
irrespective of shred exist or not ! the rule is triggering in this case

Reply:
See here mkfs utility is not there in my system. Also the rule is triggering irrespective of utility existence

![abcd](https://github.com/falcosecurity/event-generator/assets/116095646/619ff77e-264a-41c1-af49-0f807af95309)